tileview/src/main/java/com/qozix/tileview/widgets/ZoomPanLayout.java

```diff
@@ -778,9 +794,10 @@ private boolean setupZoomAnimation( float scale ) {
       return false;
     }
 
-    public void animateZoomPan( int x, int y, float scale ) {
+    public void animateZoomPan( int x, int y, float scale, @Nullable ZoomPanListener.Origination origination ) {
```

Comment:
this is a breaking change.  that's a public method.  you can add a second signature like you've described, but we can't have any breaking changes.